tests/pems/permutations/generate-certs.sh

```diff
@@ -126,20 +128,34 @@ cert-gen () {
     rm intermediate.csr
     rm client.csr
     rm ca-key.pem
+    rm ca-cert.srl
+    rm intermediate-cert.srl
 
```

Comment:
Since we are now committing these certs to the repo, can we not add the ones that aren't actually used? Some of the artifacts you are committing don't look like they are needed.